org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -79,14 +142,56 @@ public class MRTKRayInteractor :
 
         #region IHandedInteractor
 
-        Handedness IHandedInteractor.Handedness => (xrController is ArticulatedHandController handController) ? handController.HandNode.ToHandedness() : Handedness.None;
+        Handedness IHandedInteractor.Handedness
+        {
+            get
+            {
+                #pragma warning disable CS0618 // Type or member is obsolete
+                #pragma warning disable CS0612 // Type or member is obsolete
+                if (xrController != null)
+                {
+                    return (xrController is ArticulatedHandController handController) ? handController.HandNode.ToHandedness() : Handedness.None;
+                }
+                else if (HandModel != null)
+                {
+                    return HandModel.HandNode.ToHandedness();
+                }
+                else
+                {
+                    Debug.LogWarning($"Cannot determine Handedness of {name} because there is no associated HandModel.");
+                }
+                return Handedness.None; //If neither an XRController nor a HandModel is associated with this interactor then return None as handedness.
+                #pragma warning restore CS0612
+                #pragma warning restore CS0618
+            }
+        }
 
         #endregion IHandedInteractor
 
         #region IVariableSelectInteractor
 
         /// <inheritdoc />
-        public float SelectProgress => xrController.selectInteractionState.value;
+        public float SelectProgress
+        {
+            get
+            {
+                #pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Limit the scope of the warning disable...it should only cover controller usage, and nothing else.  PLease follow the pattern I used in my draft PR 

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/745/files

Reply:
done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/17305f6a751cc7715f888db852aaebace17f6dd9